app/routes/users.js

```diff
@@ -27,8 +27,10 @@ router.post('/', function(req, res) {
 
 router.post('/signin', function(req, res) {
   userService.authenticateUser(req.body.emailOrUsername, req.body.password)
-    .then((isMatching) => {
-      res.json(isMatching);
+    .then((user) => {
```

Comment:
Opted to return user object to set req.session.user. Didn't know if there was a better way to go about this.